src/main/webapp/signUp.html

```diff
@@ -1,22 +1,42 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>altruism-ator</title>
+    <title>Sign Up</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
-      <header>
-        <div class="container">
-            <nav>
-                <ul>
-                    <li class="nav-item"><a href="index.html">Home</a></li>
-                    <li class="nav-item"><a href="profile.html">Profile</a></li>
-                    <li class="nav-item"><a href="createYourOwn.html">Create your own</a></li>
-                    <li class="nav-item"><a href="chat.html">Chat</a></li>
-                </ul>
-            </nav>
-        </div>
-      </header>
+    <header>
+      <div class="container">
+        <nav>
```

Comment:
Same thought as before: I don't think we need the navbar on the sign up page.

Reply:
Agree!